backend/swagger/cirrusspark.yaml

```diff
@@ -0,0 +1,1257 @@
+swagger: "2.0"
+info:
+  description: "This is the REST diagram of Ciruss Spark API"
+  version: "1.0.0"
+  title: "Ciruss Spark"
+  termsOfService: "http://swagger.io/terms/"
+  contact:
+    email: "badobtech@gmail.com"
+  license:
+    name: "Apache 2.0"
+    url: "http://www.apache.org/licenses/LICENSE-2.0.html"
+host: "cirrus-spark-backend-490.herokuapp.com"
+basePath: "/"
+tags:
+- name: "accounts"
+  description: "API for managing accounts"
+- name: "calls"
+  description: "API for managing calls"
+- name: "loghours"
+  description: "API for managing loghours"
+- name: "project"
+  description: "API for managing projects"
+- name: "tasks"
+  description: "API for managing tasks"
+- name: "auth"
+  description: "API for authorization"
+- name: "pet"
+  description: "Everything about your Pets"
+  externalDocs:
+    description: "Find out more"
+    url: "http://swagger.io"
+- name: "store"
+  description: "Access to Petstore orders"
+- name: "user"
+  description: "Operations about user"
+  externalDocs:
+    description: "Find out more about our store"
+    url: "http://swagger.io"
+schemes:
+- "https"
+- "http"
+paths:
+  /accounts/employee:
+    post:
+      tags:
+      - "accounts"
+      summary: "Create new employee aaccount"
+      description: ""
+      operationId: "createEmployeeAccount"
+      consumes:
+      - "application/json"
+      produces:
+      - "application/json"
+      parameters:
+      - in: "body"
+        name: "body"
+        description: "Employee body"
+        required: true
+        schema:
+          $ref: "#/definitions/Pet"
+      responses:
+        "400":
+          description: "Bad request"
+        "401":
+          description: "Unauthorized"
+        "500":
+          description: "Unexpected error"
+        "201":
+          description: "Created"
+    get:
+      tags:
+      - "accounts"
+      summary: "Get existing employee account"
+      description: ""
+      operationId: "getEmployeeAccount"
+      consumes:
+      - "application/json"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "email"
+        in: "query"
+        description: "Email of employee to get"
+        required: true
+        type: "string"
+      responses:
+        "400":
+          description: "Invalid email supplied"
+        "404":
+          description: "Not found"
+        "401":
+          description: "Unauthorized"
+        "500":
+          description: "Unexpected error"
+        "200":
+          description: "OK"
+  /accounts/allEmployees:
+    post:
+      tags:
+      - "accounts"
+      summary: "Get all employee accounts"
+      description: ""
+      operationId: "getAllEmployeeAccount"
+      consumes:
+      - "application/json"
+      produces:
+      - "application/json"
+      responses:
+        "404":
+          description: "Not found"
+        "200":
+          description: "OK"
+  /accounts/allEmployees/{businessId}:
+    post:
```

Comment:
here as well

Reply:
Yes you're right. I merged with Simon's YAML so he might have made some mistakes